android/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
           package="com.flutter_webview_plugin">
-    <uses-sdk android:minSdkVersion="16" />
+
```

Comment:
I think there was PR for this to add min sdk, why was it removed?

Reply:
I removed by the suggestion of IDE, I will revert it